sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 ### Breaking Changes
 
+- Removed `azure-core-serializer-json-jackson` dependency. If a custom `JsonSerializer` is required while interact with
+  `SearchClient` or `SearchAsyncClient` please include a dependency on a library that implements `JsonSerializer`, such
+  as `azure-core-serializer-json-jackson` or `azure-core-serializer-json-gson` provided by the Azure SDKs or an external
```

Comment:
Even without including any of the dependencies, we should have the equivalent of having `azure-core-serializer-json-jackson`'s default JsonJacksonSerializer.

Reply:
Do you want me to add a statement to this that a default implementation is used based on `JacksonAdapter`'s serialization logic in `azure-core`